### PR TITLE
ARN-2310 Add user-friendly access denied page

### PR DESCRIPTION
**Pull request by devkokov**

### CODE REVIEW

**Comment by alexw-moj** [comment]
I've never tried it, but have you considered webjars? https://mvnrepository.com/artifact/org.webjars.npm/govuk-frontend/5.6.0

Curious to know if it's easier than just dropping the assets in, and if it can be renovated.

**Comment by devkokov** [comment]
> I've never tried it, but have you considered webjars? https://mvnrepository.com/artifact/org.webjars.npm/govuk-frontend/5.6.0
> 
> Curious to know if it's easier than just dropping the assets in, and if it can be renovated.

Good shout! I'll give it a go

File: src/main/resources/templates/fragments/layout.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" lang="en"
+      xmlns:th="http://www.thymeleaf.org" class="govuk-template">
+<head>
+  <meta charset="utf-8">
+  <title layout:title-pattern="$LAYOUT_TITLE - $CONTENT_TITLE"
+         th:text="${(param.error != null || error != null) ? 'Error: ' : ''} + 'Strengths and needs'">
+    Strengths and needs
+  </title>
```
Comment (Thomas-Geraghty) [inline src/main/resources/templates/fragments/layout.html:9]: Probably want to change this to be `Handover Service` too mate

File: src/main/resources/templates/fragments/layout.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" lang="en"
+      xmlns:th="http://www.thymeleaf.org" class="govuk-template">
+<head>
+  <meta charset="utf-8">
+  <title layout:title-pattern="$LAYOUT_TITLE - $CONTENT_TITLE"
+         th:text="${(param.error != null || error != null) ? 'Error: ' : ''} + 'Strengths and needs'">
+    Strengths and needs
```
Comment (Thomas-Geraghty) [inline src/main/resources/templates/fragments/layout.html:8]:
```suggestion
         th:text="${(param.error != null || error != null) ? 'Error: ' : ''} + 'Handover Service'">
    Handover Service
```

Comment (devkokov) [reply src/main/resources/templates/fragments/layout.html]: well spotted! thanks

**Review by Thomas-Geraghty — APPROVED** [review]

**Pull request merged**